src/main/cpp/src/json_utils.cu

```diff
@@ -276,7 +280,7 @@ std::pair<std::unique_ptr<cudf::column>, rmm::device_uvector<bool>> cast_strings
 
   auto output = cudf::make_fixed_width_column(
     cudf::data_type{cudf::type_id::BOOL8}, string_count, cudf::mask_state::UNALLOCATED, stream, mr);
-  auto validity = rmm::device_uvector<bool>(string_count, stream);  // intentionally not use `mr`
+  auto validity = rmm::device_uvector<bool>(string_count, stream);
```

Comment:
do we need to pass `mr` now?

Reply:
No, I just remove the comment as I thought it is not necessary. Maybe will replace it with other comments.